### DISCUSSION TITLE
Color from content

### DISCUSSION THREAD

**Discussion by Lawrightax** [material-components/material-web#4397] [Q&A] [answered]
Hello, 
I'm new to material 3 and was wondering if Dynamic Color from Content would be implemented into web and how would it work ?
I'm curious since it would be a big step for Web Development to have "dynamic" theme for a component !
Sorry if this question was asked already and thanks for the time you spend on this !

**Comment by asyncliz** [comment, marked as answer]
There is a project called [material color utilities](https://github.com/material-foundation/material-color-utilities) that provides this feature for the web. It's currently being updated to support the latest dynamic color tokens, but is a good place to start for dynamic theming.

**Reply by Lawrightax** [reply]
Oh I didn't know, I'm gonna check it out ! 
Thanks a lot for the quick response !
